lint-staged.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	'*.{js,jsx}': [ 'npm run format:js', 'eslint' ],
```

Comment:
I think we should include `ts,tsx` here just in case we ever invest in moving to TypeScript 👍

Reply:
Sounds good, fixed in a7933f65